### ISSUE TITLE
List Devices leads to frozen utility - Windows 10 x64

### ISSUE THREAD

**Issue by ibaranov-cp** [kiibohd/kii-dfu#3]
Hello,

using Kii-dfu, and selecting "List Devices" leads to greying out all options, and no further response from the program. No return codes, nothing.

However, command line dfu-util seems to detect things just fine. Gives back my serial number.

**Comment by haata** [comment]
Hmm, that's interesting.

Have you tried using administrator privileges?

**Comment by ibaranov-cp** [comment]
Yup, all tricks in the book know to me, admin mode on both util and gui,
starting in safe mode. None seemed to work. Suspect gui bug as the actual
command line util worked fine for my re-programming.

P.S hello from a university school mate Jacob.

I can poke around the keyboard code itself, and may send pull requests for
improvements

**Comment by haata** [comment]
Thanks!
The GUI code is pretty simple, but yeah I can take a look (it was written before Windows 10, so who knows...).

I remember it being pretty annoying to package all the shared libraries up (or even statically compile).
